common/pref_names.h

```diff
@@ -56,6 +56,7 @@ extern const char kNewTabPageShowStats[];
 extern const char kNewTabPageShowRewards[];
 extern const char kNewTabPageShowBinance[];
 extern const char kNewTabPageShowGemini[];
+extern const char kNewTabPageShowBitcoinDotCom[];
```

Comment:
This should go in `components/moonpay/common/moonpay_pref_names.h` instead

Reply:
moved